### PR TITLE
added in bigquery sample dashboard files.

### PR DESCRIPTION
**Pull request by philshin-google**

### CODE REVIEW

File: dashboards/bigquery/metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+id: resource_list/bigquery_dataset
```
Comment (EvanSimpson) [inline dashboards/bigquery/metadata.yaml:1]: Can you rename the directory from `bigquery` to `google-bigquery` to match the convention used by the other Google services dashboards?

File: dashboards/bigquery/metadata.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+display_name: BigQuery
+category: resource_list
+related_integrations:
+  - id: bigquery
+    platform: GCP
+spec_url: resource_list/bigquery_dataset
+canonical_url: /monitoring/dashboards/resourceList/bigquery_dataset
```
Comment (EvanSimpson) [inline dashboards/bigquery/metadata.yaml:8]: These fields aren't relevant to sample dashboards and should be removed.

File: dashboards/bigquery/metadata.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+id: resource_list/bigquery_dataset
+display_name: BigQuery
+category: resource_list
```
Comment (EvanSimpson) [inline dashboards/bigquery/metadata.yaml:3]: Category should be BigQuery

File: dashboards/bigquery/metadata.yaml (hunk 1 above)
Comment (EvanSimpson) [inline dashboards/bigquery/metadata.yaml:1]: This file should match the format used in other directories in this repository, e.g. https://github.com/GoogleCloudPlatform/monitoring-dashboard-samples/blob/master/dashboards/google-anthos/metadata.yaml

This file needs to support multiple entries, so there should be a top level `sample_dashboards` key, with a list of dashboards nested underneath.

Comment (philshin-google) [reply dashboards/bigquery/metadata.yaml]: moved files to google-bigquery.

Comment (philshin-google) [reply dashboards/bigquery/metadata.yaml]: refactored file.

File: dashboards/bigquery/metadata.yaml (hunk 3 above)
Comment (philshin-google) [reply dashboards/bigquery/metadata.yaml]: changed.

**Comment by philshin-google** [comment]
> Can you get a screenshot which has data in the table count chart?

I've tried multiple projects, backends, (unsuccessfully) manually creating tables. I'm not sure if the backends for the bigquery metrics going into the dashboard builder are functioning properly. Do you want me to contact the API owner?

**Comment by philshin-google** [comment]
> Can you get a screenshot which has data in the table count chart?

resolved!

**Review by EvanSimpson — APPROVED** [review]

**Pull request merged**